### ISSUE TITLE
@var etc doesnt work for a non public classes.

### ISSUE THREAD

**Issue by gochev** [manifold-systems/manifold#359]
**Describe the bug**

if you just define your class as internal in the same package as main the @var will not work - the compile will fail, and if you run it with the intellij plugin you will get NPE.

**To Reproduce**

 ```
import manifold.ext.props.rt.api.override;
import manifold.ext.props.rt.api.val;
import manifold.ext.props.rt.api.var;

public class Mainjava {
    public static void main(String[] args) {
        User u = new User();
        u.name = "dabe-usere"; //even this wont work.
        System.out.println(u.name); / and this
//        u.title = "neshto";

    }
}

class User {
    private String name;
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @var
    String title;
}
 ```

**Expected behavior**
To work also for classes defined like this one.

**Screenshots**
On maven compile
![image](https://user-images.githubusercontent.com/5537142/167369822-d7d4f8ed-85c4-40ad-8a02-3bd08f181abd.png)
On Run
![image](https://user-images.githubusercontent.com/5537142/167369874-cea759f2-7fd0-4cd2-9381-822a65d731c8.png)

**Desktop (please complete the following information):**
- OS Type & Version:  WSL2 / Ubuntu 
- Java/JDK version: openjdk version "11.0.12" 2021-07-20
- IDE version (IntelliJ IDEA or Android Studio): 2021.2.3
- Manifold version: 2022.1.10
- Manifold IntelliJ plugin version: 2021.2.48

**Additional context**
If I move this into a PUBLIC class like this it works
 ```
package user;

import manifold.ext.props.rt.api.var;

public class User {
    private String name;
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @var
    String title;
}
 ```

it is not because of the default package.. also tried to move the main and internal class into a package (instead of empty) its the same.

Still AMAZING work :) LOVE IT !

**Comment by rsmckinney** [comment]
Hi @gochev. It looks like your use-case involves a package class **in the same file** as the main class, right? That case appears to be broken. However, the general case of a package-private class with properties works as it should. I'll take a deeper dive into the former case.  Thanks for reporting this!

**Comment by rsmckinney** [comment]
I've verified the general case of a package-private class with properties works as it should. The only case that does not work is when the class is a top-level class that resides in another top-level  class' file. Manifold generally ignores that case everywhere; it is not likely to get fixed. Sorry.

**Comment by rsmckinney** [comment]
I suppose what should be done is to produce a compile error instead of causing an NPE.  Expect that soon.

**Comment by rsmckinney** [comment]
Fixed with release 2022.1.32
